### ISSUE TITLE
[v3] Implement new Concept Exercise: static-members

### ISSUE THREAD

**Issue by ErikSchierboom** [exercism/fsharp#842]
This issue describes how to implement the `static-members` concept exercise for the F# track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

The goal of this exercise is to teach the student the basics of the Concept of StaticMembers in F#.

## Learning objectives

- Know the difference is between static and instance members.
- Know how to define static methods.
- Know how to define static properties.
- Know how to define static let bindings.
- Know how to define static do bindings.

## Out of scope

- Operator overloading.
- `ThreadStatic` attribute.

## Concepts

The Concepts this exercise unlocks are:

- `static-members`: know the difference is between static and instance members; know how to define static methods; know how to define static properties; know how to define static let bindings; know how to define static do bindings.

## Prerequisites

This exercise's prerequisites Concepts are:

- `classes`: know how to define classes.
- `constructors`: know how to define `do` bindings.
- `properties`: know how to define properties.

Any data types used in this exercise (e.g. `strings` or `integers`) should also be added as prerequisites.

### Hints

- [Static methods][static-methods]: how to define static methods.
- [Static properties][static-properties]: how to define static properties.
- [let-bindings][let-bindings]: how to define static `let` bindings.
- [do-bindings][do-bindings]: how to define static `do` bindings.

### After

- [Static methods][static-methods]: how to define static methods.
- [Static properties][static-properties]: how to define static properties.
- [let-bindings][let-bindings]: how to define static `let` bindings.
- [do-bindings][do-bindings]: how to define static `do` bindings.

## Representer

This exercise does not require any specific representation logic to be added to the [representer][representer].

## Analyzer

This exercise does not require any specific logic to be added to the [analyzer][analyzer].

## Implementing

To implement this exercise, please [follow these instructions](https://github.com/exercism/v3/blob/master/languages/fsharp/reference/implementing-a-concept-exercise.md).

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

[how-to-implement-a-concept-exercise]: https://github.com/exercism/v3/blob/master/docs/maintainers/generic-how-to-implement-a-concept-exercise.md
[implemented-exercises]: https://github.com/exercism/v3/tree/master/languages/fsharp/exercises/concept/README.md#implemented-exercises
[analyzer]: https://github.com/exercism/fsharp-analyzer
[representer]: https://github.com/exercism/fsharp-representer
[meta-design]: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/numbers/.meta/design.md
[meta-config.json]: https://github.com/exercism/v3/blob/master/languages/fsharp/exercises/concept/numbers/.meta/config.json
[docs-concept-exercises]: https://github.com/exercism/v3/blob/master/docs/concept-exercises.md
[static-methods]: https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/members/methods#static-methods
[static-properties]: https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/members/properties#static-and-instance-properties
[let-bindings]: https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/members/let-bindings-in-classes
[do-bindings]: https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/members/do-bindings-in-classes
